.github/workflows/default.yml

```diff
@@ -0,0 +1,83 @@
+name: PHP Workflow
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: [7.4, 8.1]
```

Comment:
It's always recommended to test all codebase for multiple PHP version to make sure all are compatible back and forth